zebra-chain/src/transparent/tests/vectors.rs

```diff
@@ -37,3 +46,37 @@ fn parse_coinbase_height_mins() {
     let case4 = vec![0x04, 0x11, 0x00, 0x00, 0x00];
     assert!(parse_coinbase_height(case4).is_err());
 }
+
+#[test]
+fn get_transparent_output_address() -> Result<()> {
```

Comment:
Coverage: Can we add a test that gets the addresses from every test vector in `MAINNET_BLOCKS` and `TESTNET_BLOCKS`?

We could assert that every transparent output has an address, if the test vectors all have valid addresses.

This also checks for performance, panics, or crashes.